syntax = "proto3";

import "google/protobuf/any.proto";

package opentera.protobuf;

message ServerCommand {

    enum CommandType
    {
        REGISTER_DEVICE = 0;
    }
    CommandType type = 1;
    google.protobuf.Any data = 2;

}